pyrobosim/pyrobosim/core/world.py

```diff
@@ -1784,7 +1808,13 @@ def collides_with_robots(self, pose: Pose, robot: Robot | None = None) -> bool:
                 return True
         return False
 
-    def is_path_collision_free(self, path: Path, step_dist: float = 0.01) -> bool:
+    def is_path_collision_free(
```

Comment:
Same comment here re: whether at this point it's easier to just make this a utility function that accepts the world and the robot